### PR TITLE
updated yield optimizers based on discord feedback

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

**Comment by johnnyonline** [comment]
- protocol/yield-optimizers/README.md looks good.
- protocol/yield-optimizers/compounders.md looks good.
- protocol/yield-optimizers/concentrators.md is not 100% accurate I think.

concentrators.md: 
1. "... provide the user with minimum exposure to reward tokens" - this is technically correct, but doesnt communicate the more significant value of this strategy. The Concentrator's point is to actually minimize the exposure to the _vault assets_. Why would someone want to hold an asset and minimize his exposure to it at the same time? Maybe because those assets generate great yield, but they have certain attributes that the investor would like to minimize exposure to, like IL, or not enough exposure to some other asset the investor is more interested in.
2. "... Vault shares simply represent receipts to the auto-compounding vault assets that are managed by the vaults. Vault shares increase in value at every compounding event, as more of the vault assets accumulate, while the amount of vault shares remain unchanged." - In the Concentrators, Vault shares represent a pro-rata claim on the principal deposited (vault assets). The shares value is always equal to the vault assets, as rewards are invested into some other vault, the vault assets to shares ratio is always the same. 
3. Also need to mention that while holding the Concentrator fortTokens (shares), the holder is accumulating rewards which can be claim at any time, even if the shares are not in the position of the investor anymore.

File: protocol/yield-optimizers/concentrators.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+
+Concentrators mint ERC20 vault shares (fortTokens) for users upon deposits. In
+turn, Concentrators burn these vault shares again upon withdrawals. Vault shares
+simply represent receipts to the auto-compounding vault assets that are managed
+by the vaults. Vault shares increase in value at every compounding event, as
+more of the vault assets accumulate, while the amount of vault shares remain
+unchanged.
+
+Users may chose to utilize fortTokens in different DeFi Protocols or Fortress
+Strategies for additional capital efficiency.
```
Comment (xh3b4sd) [inline protocol/yield-optimizers/concentrators.md:23]:
```suggestion
Users may chose to utilize FortTokens in different DeFi Protocols or Fortress
Strategies for additional capital efficiency.
```

File: protocol/yield-optimizers/concentrators.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
 # Concentrators
 
-Concentrators re-direct earnings into different strategies creating structured yield pipelines.
+Fortress auto-concentrators (Concentrators) are ERC-4626 vaults that use certain
+farming strategies to provide the user with minimum exposure to reward tokens.
```
Comment (xh3b4sd) [inline protocol/yield-optimizers/concentrators.md]:
```suggestion
farming strategies to provide the user with minimum exposure to vault assets. 
```

**Comment by xh3b4sd** [comment]
Please check again @johnnyonline. Tried to incorporate changes as discussed. Let me know if I got something wrong. Feel free to approve when all looks good.

File: protocol/yield-optimizers/concentrators.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
 # Concentrators
 
-Concentrators re-direct earnings into different strategies creating structured yield pipelines.
+Fortress auto-concentrators (Concentrators) are ERC-4626 vaults that use certain
+farming strategies to provide the user with minimum exposure to vault assets. 
+
+Concentrators claim their strategy rewards periodically and swap their reward
+tokens into target assets, which are then invested into vaults that provide risk
+profiles different from the originating vault. That is how Concentrators
+minimize exposure to reward tokens, by not holding on to them, but repurposing
```
Comment (johnnyonline) [inline protocol/yield-optimizers/concentrators.md]:
```suggestion
minimize exposure to reward tokens, by not holding on to them, but repurposing
```

this whole section is problematic, maybe more like "That is how Concentrators leverage the investment attributes of the principal, while not leaving the entire position exposed to the risks of the initial investment"

Comment (johnnyonline) [reply protocol/yield-optimizers/concentrators.md]: other than that looks good to me

**Pull request merged**